### PR TITLE
Customisable move/resize window mouse buttons

### PR DESCRIPTION
**Pull request by Sidd-Dino**
Addressing the issue #189.

Two new options added to berry and berryc
```sh
berryc move_button 1-5
berryc resize_button 1-5
```
---
**Buttons Tested**
```sh
1 - Left Mouse Button
2 - Scroll Button (Press Down on Scroll Wheel)
3 - Right Mouse Button
```

### CODE REVIEW

**Comment by Sidd-Dino** [comment]
The modifications I've added also allows me to address the issue #117 .
I haven't fixed it yet but I can if you wish @JLErvin

File: wm.c [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 #include "config.h"
 
+#include <X11/X.h>
```
Comment (JLErvin) [inline wm.c:6]: why is this needed?

Comment (Sidd-Dino) [reply wm.c:6]: welp my bad
must hae been my ide auto adding libraries
I'll clean it up

Comment (Sidd-Dino) [reply wm.c:6]: Welp my bad
Must have been vscode adding libraries on it's own

**Comment by Sidd-Dino** [comment]
I've made the changes mentioned. 
Is there anything else I need to do?

**Review by JLErvin — APPROVED** [review]
